src/ProjectNeon/Assets/Features/Battle/Card.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System.Collections.Generic;
-using UnityEngine;
-
-public class Card : ScriptableObject
-{
-    public Sprite Art;
-    public string Description;
-    public string TypeDescription;
-    public List<CardAction> actions;
-    public Scope scope;
-    public Group group;
-}
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class Card : ScriptableObject
```

Comment:
This one is a pure definition `ScriptableObject`, he must live in `Data/Templates`